doc/api/util.md

```diff
@@ -620,8 +620,35 @@ util.inspect(obj);
 added: v6.6.0
 -->
```

Comment:
Oh, almost forgot: Can you add a `changes:` block here that describes the change in a succinct fashion? (compare e.g. the metadata for `crypto.createCipher` in `crypto.md`)

Reply:
Like [this](https://github.com/nodejs/node/pull/20857/commits/fd0cbeb0ccd79bcd9e82eb4813c91a59bd2ba9c4)?

Reply:
Yup, looks good!